docker-for-mac/index.md

```diff
@@ -156,10 +160,16 @@ File share settings are:
 - **Apply & Restart** makes the directory available to containers using Docker's
   bind mount (`-v`) feature.
 
-  There are some limitations on the directories that can be shared:
+The following paths will always be shared from Linux, rather than the Mac host:
```

Comment:
I don’t understand what this means.

Reply:
I'm in two minds about whether to try to include (and explain) this. I'm trying to say that `docker run -v /dev:/dev` will share the `/dev` in the helper VM, not on the host. Some things like `/dev/zero` will work but obviously `/dev/sda` might not, depending on what your expectations were. Maybe I'll remove it.